packages/@azure/servicebus/data-plane/lib/session/messageSession.ts

```diff
@@ -523,7 +532,7 @@ export class MessageSession extends LinkEntity {
     }
     if (!options) options = {};
     this._isReceivingMessages = true;
-    this.maxConcurrentCallsPerSession = 1;
+    this.maxConcurrentCallsPerSession = options.maxConcurrentCallsPerSession != undefined ? options.maxConcurrentCallsPerSession : 1;
```

Comment:
The check can be more strict here.. Let's check for the type of `options.maxConcurrentCallsPerSession` and use it only if the type is a number and is > 0

Same for `maxConcurrentCalls` in `messageReceiver.ts` file.

Reply:
fixed